patroni/ctl.py

```diff
@@ -128,17 +130,28 @@ def auth_header(config):
         return {'Authorization': 'Basic ' + base64.b64encode(config['restapi']['auth'].encode('utf-8')).decode('utf-8')}
 
 
-def request_patroni(member, request_type, endpoint, content=None, headers=None):
+def request_patroni(member, request_type, endpoint, content=None, headers=None, insecure=False):
     headers = headers or {}
     url_parts = urlparse(member.api_url)
     logging.debug(url_parts)
     if 'Content-Type' not in headers:
         headers['Content-Type'] = 'application/json'
 
     url = '{0}://{1}/{2}'.format(url_parts.scheme, url_parts.netloc, endpoint)
-
+    # Retrieve certfile from configfile in restapi section if any
+    cert = (click.get_current_context().obj.get('restapi', {}).get('certfile'),
```

Comment:
Doc http://docs.python-requests.org/en/master/user/advanced/#client-side-certificates tells that `cert` is used for client-side certificates, but here you are presumably trying to use server-side.

Alright, let's assume that this is a special patronictl config. Even in this case there is a problem: `requests` module expect to get either a `string` (filename containing cert+key) or a tuple. Not sure how it will work if it will get for example `('cert', None)`, because apparently [keyfile](https://patroni.readthedocs.io/en/latest/SETTINGS.html?highlight=keyfile#rest-api) is optional: https://docs.python.org/2/library/ssl.html#ssl.SSLContext.load_cert_chain

Reply:
Indeed, I will remove this part.